applications/openshift/api-server/api_server_auth_mode_node/rule.yml

```diff
@@ -0,0 +1,38 @@
+prodtype: ocp4
+
+title: Must have authorization-mode Node set
+
+description: 'Restrict kubelet nodes to reading only objects associated with them.'
+
+rationale: |-
+  The Node authorization mode only allows kubelets to read Secret,
+  ConfigMap, PersistentVolume, and PersistentVolumeClaim objects
+  associated with their nodes.
+
+severity: medium
+
+references:
+    cis: 1.2.8
+
+ocil_clause: 'The Node authorization-mode is configured and enabled on the master node'
+
+ocil: |-
+    Run the following command on the master node(s):
+    <pre>$ oc get configmap config -n openshift-kube-apiserver -ojson | jq -r '.data["config.yaml"]' | grep '"authorization-mode":\[.*Node"\]'</pre>
+    The output should show that the "authorization-mode" list contains the "Node" authorizer.
+
+warnings:
+- general: |-
+    {{{ openshift_cluster_setting("/api/v1/namespaces/openshift-kube-apiserver/configmaps/config") | indent(4) }}}
+
+template:
+  name: yamlfile_value
+  vars:
+    ocp_data: "true"
+    entity_check: "at least one"
+    filepath: /api/v1/namespaces/openshift-kube-apiserver/configmaps/config
+    yamlpath: '.data["config.yaml"]'
+    values:
+    - value: '"authorization-mode":\[.*Node"\]'
```

Comment:
This construct is dependable on the order of the values, isn't it? I propose slightly different regex `'"authorization-mode":\[[^]]*"Node"'`

Reply:
I'm not acquainted with this notation: `\[[^]]*` what's that?

Reply:
thanks! added that